test/kitchen/test/integration/system-probe-test/rspec_datadog/system-probe-test_spec.rb

```diff
@@ -82,6 +82,7 @@ def get_timeout(package)
 
     base_env = {
       "DD_SYSTEM_PROBE_BPF_DIR"=>"#{tests_dir}/pkg/ebpf/bytecode/build",
+      "DD_SYSTEM_PROBE_JAVA_DIR"=>"/tmp/system-probe-tests/pkg/network/java",
```

Comment:
Shouldn't it be like that?
```suggestion
      "DD_SYSTEM_PROBE_BPF_DIR"=>"#{tests_dir}/pkg/ebpf/bytecode/build",
      "DD_SYSTEM_PROBE_JAVA_DIR"=>"#{tests_dir}/pkg/network/java",
```

Reply:
yes rebase failed :(